bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,254 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+import json
+
+
+@linter(executable='csscomb',
+        output_format='corrected',
+        use_stdout=True,
+        use_stdin=False,
+        result_message='Change coding style based on CSS rules written in ' +
+                       '.csscomb.json',
+        )
+class CSSCombBear:
+    """
+    CSScomb is a coding style formatter for CSS.
+
+    For more information, consult <https://github.com/csscomb/csscomb.js>.
+    """
+
+    LANGUAGES = {"CSS"}
+    REQUIREMENTS = {NpmRequirement('csscomb')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/bxpke7nizyxdf6mlx6ss4h405'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return (filename,)
+
+    def generate_config(filename, file,
+                        verbose: bool = True,
+                        remove_empty_rulesets: bool = True,
```

Comment:
uhm, I'd rather make it `allow_empty_rulesets`, the fix is always an optional thing and the user configures from the analysis point of view.